crates/oxc_linter/src/rules/unicorn/no_array_for_each.rs

```diff
@@ -114,12 +114,12 @@ fn test() {
         r"1?.forEach((a, b) => call(a, b))",
         r"array.forEach((arrayInArray) => arrayInArray.forEach(element => bar(element)));",
         r"array.forEach((arrayInArray) => arrayInArray?.forEach(element => bar(element)));",
-        r"array.forEach((element, index = element) => {})",
+        r"array.forEach((element, index = element1) => {})",
```

Comment:
This shouldn't trigger a duplicate parameter error, shall we revert the change?

Reply:
Yes, we need to revert